configure.py

```diff
@@ -1566,7 +1566,6 @@ def main():
 
   if is_macos():
     environ_cp['TF_NEED_TENSORRT'] = '0'
-    environ_cp['TF_ENABLE_XLA'] = '0'
```

Comment:
Please also remove the similar line for windows (line 1563).

Reply:
Done, thanks